roles/install-k8s/tasks/main.yml

```diff
@@ -24,6 +24,8 @@
     src: kubelet.service.j2
     dest: /usr/lib/systemd/system/kubelet.service
     mode: '0644'
+  vars:
+    KUBELET_KUBECONFIG_PERF_ARGS: "{{ lookup('env','KUBELET_CONFIG_PERF_ARGS') | default('', True) }}"
```

Comment:
you can rename like `KUBELET_EXTRA_ARGS` and see we can use that existing environment in the template file.

Reply:
@mkumatag done